resources/js/visual-editor/site-editor/styles/styles-navigator.css

```
@@ -0,0 +1,96 @@
+    background: transparent;
+    border: none;
+    border-radius: 0.25rem;
+    color: inherit;
+    cursor: pointer;
+    font: inherit;
+    padding: 0.375rem 0.5rem;
+    text-align: left;
+}
+
+.ap-site-editor__styles-navigator-link:hover {
+    background: color-mix(in srgb, currentColor 6%, transparent);
```

Stylelint (17.9.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)